php/ad-servers/class-ad-layers-dfp.php

```diff
@@ -300,11 +300,30 @@ public function get_settings_fields() {
 					'code' => new Fieldmanager_Textfield( __( 'Code', 'ad-layers' ) ),
 					'path_override' => new Fieldmanager_TextField( __( 'Custom Path Template', 'ad-layers' ) ),
 					'sizes' => new Fieldmanager_Group( array(
-						'limit' => 0,
+						'add_more_label' => __( 'Add Size', 'ad-layers' ),
+						'description' => __( 'Size cannot be 0x0.', 'ad-layers' ),
+						'group_is_empty' => function( $values ) {
+							if ( isset( $values['width'], $values['height'] ) ) {
```

Comment:
You should just be able to do

```php
return empty( $values['width'] ) && empty( $values['height'] );
```

Reply:
Also, if you really need all of these checks, then it might be easier to read stacked in a single if statement